src/germ_udpsrvr.c

```diff
@@ -23,7 +23,7 @@
 
 
 //event data buffer for a frame
-uint16_t evtdata[5000000000];
+uint16_t evtdata[1000000000];
```

Comment:
I segfaulted when this was 5 times larger. not sure completely why yet... but will test detector first